src/ManojN/Q2arrayDuplicateNumber.java

```diff
@@ -0,0 +1,24 @@
+//Q2) Find duplicate number on Integer array in Java?
+
+package a2;
+
+public class Q2arrayDuplicateNumber {
+
+	static void m1(int a2[]){
+		//int a2 [] = {0, 3, 1, 2, 3};
+		for (int i=0; i<a2.length; i++){
+			
+			for(int j=i+1;j<a2.length; j++){
+				if(a2[i]==a2[j]){
+					System.out.println("missing no in array : " + a2[j]);
```

Comment:
Here this will print 3 three times if i add one more 3..
so it's wrong..
and it should print 3 as duplicate and you were printing 3 as missing number... correct it....